src/Symfony/Bundle/FrameworkBundle/DataCollector/HttpClientDataCollector.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\DataCollector;
```

Comment:
should be in the component, as done for other components

Reply:
Thanks! You're right!